plugins/plugins-types.d.ts

```diff
@@ -96,7 +96,7 @@ type DefaultPlugins = {
     useMqs?: string[];
     /**
      * Pseudo-classes and elements to use. An empty string indicates all
-     * all non-pseudo-classes and elements.
+     * are non-pseudo-classes and elements.
```

Comment:
```suggestion
     * non-pseudo-classes and elements.
```

Pretty sure this one should omit the `all` rather than replace it.